Dockerfile

```diff
@@ -0,0 +1,24 @@
+ARG PYTHON_V=3.8
+
+FROM python:${PYTHON_V}-slim AS base
+
+WORKDIR /app
+
+COPY . .
```

Comment:
The COPY command copies everything from the current directory to the `/app` directory in the container. Ensure that your build context is set correctly to avoid copying unnecessary files.